packages/create-sitecore-jss/src/templates/nextjs/src/lib/middleware/index.ts

```diff
@@ -19,7 +20,15 @@ export default async function middleware(
 ): Promise<NextResponse> {
   const response = NextResponse.next();
 
-  return (Object.values(plugins) as MiddlewarePlugin[])
+  debug.common('next middleware start');
```

Comment:
Most (if not all) middleware has specific debug namespaces with performance metrics included in the logs now. Won't these common middleware messages be extra noise in most cases?

Reply:
The idea was to get the total metric for next middleware, if we will remove that, it requires calculating it

Reply:
Ah, I missed that point. That makes sense.